vuepress/docs/docs/README.md

```diff
@@ -139,20 +140,24 @@ You can use `.rename` to change the names of the generated root types.
 
 Creating mutations is the same as queries, except you pass them as the second argument to `RootResolver`:
 
-```scala
+```scala mdoc:nest:silent
+import zio.Task
+
 case class CharacterArgs(name: String)
 case class Mutations(deleteCharacter: CharacterArgs => Task[Boolean])
-val mutations = Mutations(???)
+val mutations = Mutations(_ => ???)
 val api = graphQL(RootResolver(queries, mutations))
 ```
 
 ## Subscriptions
 
 Similarly, subscriptions are passed as the third argument to `RootResolver`:
 
-```scala
+```scala mdoc:nest:silent
+import zio.stream.ZStream
+
 case class Subscriptions(deletedCharacter: ZStream[Any, Nothing, Character])
-val subscriptions = Subscriptions(???)
+val subscriptions = Subscriptions(ZStream())
```

Comment:
Doesn't it compile with `???`?

Reply:
Since it's the last section, we can use `compile-only` trick